tests/sentry/grouping/test_fingerprinting.py

```diff
@@ -184,3 +191,143 @@ def dump_variant(v):
             },
         }
     )
+
+
+@pytest.fixture
+def default_bases():
+    return ["nextjs@2023-12-22"]
```

Comment:
@vartec do we actually need to date the files? I would think it would make sense to just have one file per SDK/platform

Reply:
Good question, I basically copied that patter from stacktrace Enhancements, where we kinda have one per strat, but still date versioned. 
Kinda makes sense with Strategies, which are also date versioned.